src/components/Trends/TrendsTable/TrendsTable.js

```diff
@@ -215,10 +216,11 @@ class TrendsTable extends PureComponent {
     const tableData = trendWords.map(({ word, score }, index) => {
       const [oldVolume = 0, newVolume = 0] = volumeChange[word] || []
       const isWordSelected = selectedTrends.has(word)
+      const isLimitReached = selectedTrends.size > 4 && !isWordSelected
```

Comment:
Without a context it's really hard to identify "what limit is reached" just by reading the variable name. Proposed name: `hasMaxWordsSelected`